reference/catalogue.yaml

```
@@ -762,6 +762,9 @@ components:
           description: The locations which provide access to the image
           items:
             $ref: '#/components/schemas/DigitalLocation'
+        aspectRatio:
+          type: float
```

I think OpenAPI has [opinions](https://swagger.io/docs/specification/data-models/data-types/#numbers) about how to specify these types
```suggestion
          type: number
          format: float
```